composite-aeson-path/package.nix

```diff
@@ -0,0 +1,16 @@
+{ mkDerivation, aeson-better-errors, base, composite-aeson, hpack
+, mtl, refined, stdenv
+}:
+mkDerivation {
+  pname = "composite-aeson-path";
+  version = "0.7.2.0";
+  src = ./.;
+  libraryHaskellDepends = [
+    aeson-better-errors base composite-aeson mtl refined
+  ];
+  libraryToolDepends = [ hpack ];
+  prePatch = "hpack";
+  homepage = "https://github.com/ConferOpenSource/composite#readme";
+  description = "composite-aeson support for Refined from the refined package";
```

Comment:
this file should be derived from the package.yaml by way of the cabal file and cabal2nix, which can be triggered via `make`. it looks like this may have been manually copied? the description doesn't match the package.yaml